app/invocation/invocation.tsx

```diff
@@ -31,16 +31,18 @@ import InvocationActionCardComponent from "./invocation_action_card";
 import TargetsComponent from "./invocation_targets";
 import { BuildBuddyError } from "../util/errors";
 import UserPreferences from "../preferences/preferences";
-import { eventlog } from "../../proto/eventlog_ts_proto";
+import Long from "long";
 import capabilities from "../capabilities/capabilities";
 import CacheRequestsCardComponent from "./cache_requests_card";
+import rpc_service from "../service/rpc_service";
 
 interface State {
   loading: boolean;
   inProgress: boolean;
   error: BuildBuddyError | null;
 
   model: InvocationModel;
+  attempt: Long;
```

Comment:
nit: is it possible to avoid using `Long` here and use `number` instead? I'd like to avoid Long unless we really need it, since it's protobufjs-specific and eventually we might want to migrate away from it since it has a bunch of issues.

Reply:
sgtm